README.md

```diff
@@ -76,6 +76,29 @@ npm run develop
 
 The app will be running at [http://localhost:3000](http://localhost:3000) or with browsersync at [http://localhost:3001](http://localhost:3001).
 
+### Clearing cached reference data
```

Comment:
Just thinking about the placement of this. @merlinc did a bit of work to split out our documentation and try and clean up the base readme. I wonder if this should live somewhere in the [docs folder](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/tree/master/docs), maybe `development.md` or `releasing.md`?

Reply:
Moved this to releasing.md